java/format/pom.xml

```diff
@@ -41,6 +41,14 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>${maven-compiler-plugin.version}</version>
+        <configuration>
+          <useIncrementalCompilation>false</useIncrementalCompilation>
```

Comment:
I'm not sure we want to do this? We should just document the right way to do a "true" clean build.

Reply:
Ah yes, I am still addressing the reviews. I still didn't find a better way to test this out.

Reply:
Finally! 
```bash
mvn clean install -Ddevelocity.cache.local.enabled=false -Ddevelocity.cache.remote.enabled=false
```

Reply:
Can we add this to the dev docs?

Reply:
Of course I will.

Reply:
Documented here.